server/src/test/java/keywhiz/auth/ldap/LdapAuthenticatorTest.java

```diff
@@ -27,19 +27,21 @@
 import java.util.Optional;
 import keywhiz.auth.User;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
-import org.junit.runner.RunWith;
+//import org.junit.runner.RunWith;
```

Comment:
Nit, kill this line? (and the other ones)

Reply:
@mcpherrinm can you add TODOs here for the commented imports?